xclim/indices.py

```diff
@@ -534,6 +566,8 @@ def tx_mean(tasmax, freq='YS'):
     return arr.mean(dim='time')
 
 
+@with_attrs(standard_name='tx_min', long_name='Minimum of daily maximum temperature',
+            cell_methods='time: minimum within {freq}')
```

Comment:
I believe this is the source of Python2 errors. I don't think that fstrings with keywords can be passed within function arguments. Do you know of a workaround?

Reply:
Confirming this. Hard coding 'years' in `cell_methods` passes py27 build.